can/util.py

```diff
@@ -188,6 +188,19 @@ def __iter__(self):
             yield m
 
 
+def set_logging_level(level_name=None):
+    """Set the logging level for python-can.
+    Expects one of: 'critical', 'error', 'warning', 'info', 'debug', 'subdebug'
+    """
+    try:
+        log.setLevel(getattr(logging, level_name.upper()))
```

Comment:
Will this work? Previously the log level was set for the 'can' logger so that it was the default for all logger under that package. Now it is set for 'can.util' so I don't think it will have much effect.